Contest_API.md

```diff
@@ -1571,65 +1571,74 @@ If the client fails to respond to multiple requests over a period of
 time (configured for each contest), it will be assumed deactivated and
 automatically removed from future callbacks.
 
-The following endpoint is associated with the webhook:
+The following endpoints is associated with the webhook:
 
-| Endpoint   | Mime-type        | Required? | Source @WF | Description                            |
-| ---------- | ---------------- | --------- | ---------- | -------------------------------------- |
-| `/webhook` | application/json | yes       | CCS        | List or register for webhook callbacks |
+| Endpoint         | Mime-type        | Required? | Source @WF | Description                                                                                                          |
+| ---------------- | ---------------- | --------- | ---------- | -------------------------------------------------------------------------------------------------------------------- |
+| `/webhooks`      | application/json | yes       | CCS        | JSON array of all webhook callbacks with elements as defined in the table below. Also used to register new webhooks. |
+| `/webhooks/<id>` | application/json | yes       | CCS        | JSON object of a single webhook callback with elements as defined in the table below                                 |
 
-JSON elements of webhook objects:
+JSON elements of webhook callback objects:
 
-| Name | Type   | Required? | Nullable? | Description           |
-| ---- | ------ | --------- | --------- | --------------------- |
-| url  | string | yes       | no        | The url for callbacks |
+| Name       | Type            | Required? | Nullable? | Description                                                                           |
+| ---------- | --------------- | --------- | --------- | ------------------------------------------------------------------------------------- |
+id           | ID              | yes       | no        | identifier of the webhook.                                                            |
+url          | string          | yes       | no        | The URL to post HTTP callbacks to.                                                    |
+endpoints    | array of string | yes       | no        | Names of endpoints to receive callbacks for. Empty array means all endpoints.         |
```

Comment:
How are endpoints specified? Which endpoints are allowed? I.e. what are the possible values here?

Sam as the old list for the event feed? (When did we remove that one? I don't remember)

Reply:
I'd say same list yes. Let's specify this in a general place? Maybe separate from this PR?

Reply:
I will re-add the list from the old wiki and add `/webhooks/<id>` to it.